string base_link
string distal_link
string control_mode
string state
bool use_body_jacobian
float64 max_vel_lin
float64 max_vel_ang
float64 max_acc_lin
float64 max_acc_ang
